atommic/collections/segmentation/nn/base.py

```diff
@@ -77,6 +79,8 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # Set threshold for segmentation classes. If None, no thresholding is applied.
         self.segmentation_classes_thresholds = cfg_dict.get("segmentation_classes_thresholds", None)
         self.segmentation_activation = cfg_dict.get("segmentation_activation", None)
+        self.segmentation_type = cfg_dict.get("segmentation_type", "MLS")
```

Comment:
@TimPaquaij same here as in rs. Maybe it's obvious, but now self.segmentation_type = cfg_dict.get("segmentation_type", "MLS") why do we need to set it to mls by default?